tests/labhub_test.py

```diff
@@ -180,6 +180,27 @@ def test_assign_cmd(self):
 
         testbot.assertCommand(cmd.format('coala', 'a', '23'),
                               'You\'ve been assigned to the issue')
+        
+        # no assignee, newcomer, initiatives/gci
+        mock_maint_team.is_member.return_value = False
+        mock_dev_team.is_member.return_value = False
+        mock_issue.labels = 'initiatives/gci',
+        mock_issue.assignees = tuple()
+        testbot.assertCommand(cmd.format('coala', 'a', '23'),
+                              'You are not eligible to be assigned\
+                               to this issue')
+        testbot.pop_message()
+
+        # no assignee, developer, initiatives/gci
+        mock_maint_team.is_member.return_value = False
+        mock_dev_team.is_member.return_value = True
+        mock_issue.labels = 'initiatives/gci',
+        mock_issue.assignees = tuple()
+        testbot.assertCommand(cmd.format('coala', 'a', '23'),
+                              'You are not eligible to be assigned\
+                               to this issue')
```

Comment:
@EverWinter23 use
```
 testbot.assertCommand(cmd.format('coala', 'a', '23'),
                                  'You are not eligible to be assigned ' \
                                  'to this issue')
```

Reply:
@nvzard @Makman2 Done

Reply:
@EverWinter23 not really. You did a silly mistake. Make the builds happy ;)